packages/hub/graphql.config.yaml

```diff
@@ -0,0 +1,5 @@
+schema: ./schema.graphql
+documents:
+  - src/**/*.tsx
+  - src/**/*.ts
+  - test/**/*.ts
```

Comment:
Note: this is mostly for VS Code extension. And it doesn't work that well yet. (https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql that I use is not aware of Relay-specific directives, so it complains about some Relay queries).